business_rules/utils.py

```diff
@@ -120,6 +124,28 @@ def check_params_valid_for_method(method, given_params, method_type_name):
             ', '.join(invalid_params), method_type_name, method.__name__))
 
 
+def check_for_default_value_for_missing_params(missing_params, method_params):
+    """
+    :param missing_params: Params missing from Rule
+    :param method_params: Params defined on method, which could have default value for missing param
+    [{
+     'label': 'release voucher from template',
+     'name': 'voucher_template_id',
+     'fieldType': 'numeric',
+     'defaultValue': 123
+    },
+    ...
+    ]
+    :return Updated missing parameters:
+    """
+    if method_params:
+        for param in method_params:
+            if param['name'] in missing_params and param.get('defaultValue'):
+                missing_params.remove(param['name'])
```

Comment:
It'd be better if you just collect the params and then return them. Then the function doesn't need to change `missing_params` but the caller function will do it.